dictionary_auto_to_en.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You should not be creating these datasets. Please add them to the test instance and call them using the config file like so: https://github.com/Datatamer/tamr-toolbox/blob/develop/tests/mocking/resources/toolbox_test.yaml#L41

Reply:
And https://github.com/Datatamer/tamr-toolbox/blob/develop/tests/enrichment/test_dictionary.py#L20